applications/ipc_radio/README.rst

```diff
@@ -76,14 +76,17 @@ The Bluetooth Low Energy and IEEE 802.15.4 functionalities can operate simultane
 Sysbuild Kconfig options
 ========================
 
-To enable the firmware, use the sysbuild configuration of :kconfig:option:`SB_CONFIG_NRF_DEFAULT_IPC_RADIO`.
+To enable the firmware, use the sysbuild configuration :kconfig:option:`SB_CONFIG_NETCORE_IPC_RADIO`.
```

Comment:
```suggestion
To enable the firmware, use the sysbuild configuration ``SB_CONFIG_NETCORE_IPC_RADIO``.
```
Format the SB Kconfig options with double backticks until we have a better solution that makes these linkable.